docs/core/diagnostics/dotnet-counters.md

```diff
@@ -154,48 +166,98 @@ dotnet-counters monitor [-h|--help] [-p|--process-id] [--refreshInterval] [count
 
   The number of seconds to delay between updating the displayed counters
 
-- **`counter_list <COUNTERS>`**
+- **`--counters`**
+
+  A comma-separated list of counters. Counters can be specified `provider_name[:counter_name]`. If the `provider_name` is used without a qualifying list of counters, then all counters from the provider are shown. To discover provider and counter names, use the [dotnet-counters list](#dotnet-counters-list) command.
 
-  A space separated list of counters. Counters can be specified `provider_name[:counter_name]`. If the `provider_name` is used without a qualifying `counter_name`, then all counters are shown. To discover provider and counter names, use the [dotnet-counters list](#dotnet-counters-list) command.
+ **`-- <command>` (for target applications running .NET 5.0 or later only)**
+
+  Prefixing the `collect` command with `--` followed by a command executes the given command as a child process and starts collecting metric from the target process. This is useful for getting metrics from runtime startup (before `Main` runs) and can be used to diagnose or monitor issues that happen early in the application startup.
```

Comment:
Hm. Not sure if prefixing is the right word here. Probably we should be consistent in the places where we document this.